File: EarlyWarningAndroid/app/src/main/java/cc/bitky/warningsystem/bean/CloudAreaItem.java
package cc.bitky.warningsystem.bean;

import cn.bmob.v3.BmobObject;
import cn.bmob.v3.datatype.BmobDate;

public class CloudAreaItem extends BmobObject {
  /**
   * 地点名
   */
  private String name;
  /**
   * 状态详情
   */
  private String detail;
  /**
   * 当前监测状态
   */
  private Boolean status;
  private Boolean enabled;
  /**
   * 生成时间
   */
  private BmobDate time;
  /**
   * 图片地址
   */
  private String photoUri;
  /**
   * 反演进度
   */
  private Integer processBar;

  private String coordinate;

  public String getCoordinate() {
    return coordinate;
  }

  public void setCoordinate(String coordinate) {
    this.coordinate = coordinate;
  }


  public CloudAreaItem(String name, BmobDate time) {
    this.name = name;
    this.time = time;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getDetail() {
    return detail;
  }

  public void setDetail(String detail) {
    this.detail = detail;
  }

  public Boolean getStatus() {
    return status;
  }

  public void setStatus(Boolean status) {
    this.status = status;
  }

  public BmobDate getTime() {
    return time;
  }

  public void setTime(BmobDate time) {
    this.time = time;
  }

  public String getPhotoUri() {
    return photoUri;
  }

  public void setPhotoUri(String photoUri) {
    this.photoUri = photoUri;
  }

  public Integer getProcessBar() {
    return processBar;
  }

  public void setProcessBar(Integer processBar) {
    this.processBar = processBar;
  }

  public Boolean getEnabled() {
    return enabled;
  }

  public void setEnabled(Boolean enabled) {
    this.enabled = enabled;
  }
}

File: SafetyWarningSystemNET/flashresUniversalSystem/cloud/bean/CloudAreaItem.cs
using cn.bmob.io;
using System;

namespace bitkyFlashresUniversal.cloud.bean
{
    public class CloudAreaItem : BmobTable
    {
        /**
         * 地点名
         */
        public String name { get; set; }
        /**
         * 状态详情
         */
        public String detail { get; set; }
        /**
         * 当前监测状态
         */
        public BmobBoolean status { get; set; }
        public BmobBoolean enabled { get; set; }
        /**
         * 生成时间
         */
        public BmobDate time { get; set; }
        /**
         * 图片地址
         */
        public String photoUri { get; set; }
        /**
         * 反演进度
         */
        public BmobInt processBar { get; set; }

        public BmobInt processStatus { get; set; }

        public String coordinate { get; set; }
        //读字段信息
        public override void readFields(BmobInput input)
        {
            base.readFields(input);
            name = input.getString("name");
            detail = input.getString("detail");
            status = input.getBoolean("status");
            enabled = input.getBoolean("enabled");
            time = input.getDate("time");
            photoUri = input.getString("photoUri");
            processBar = input.getInt("processBar");
            processStatus = input.getInt("processStatus");
            coordinate = input.getString("coordinate");
        }

        //写字段信息
        public override void write(BmobOutput output, bool all)
        {
            base.write(output, all);

            output.Put("name", name);
            output.Put("detail", detail);
            output.Put("status", status);
            output.Put("enabled", enabled);
            output.Put("time", time);
            output.Put("photoUri", photoUri);
            output.Put("processBar", processBar);
            output.Put("processStatus", processStatus);
            output.Put("coordinate", coordinate);
        }
    }
}